src/AppInstallerCLICore/Workflows/InstallFlow.cpp

```diff
@@ -269,6 +269,13 @@ namespace AppInstaller::CLI::Workflow
                 (isUpdate ? MSStoreUpdate : MSStoreInstall);
             break;
         case InstallerTypeEnum::Portable:
+            if (isUpdate && installer.UpdateBehavior == UpdateBehaviorEnum::UninstallPrevious)

```

Comment:
I think this might be redundant? Isn’t the manifest validation that users can’t set upgradeBehavior for Portable type installers, and they will always be handled as uninstallPrevious

Reply:
I was mistaken; I was thinking of the scope property, which isn’t able to be specified in the manifest for Portable installer type